packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -82,21 +79,29 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 					'is-paid': ! suggestion.is_free,
 				} ) }
 			>
-				{ suggestion.is_free ? (
-					__( 'Free' )
-				) : (
-					<>
-						<span className="domain-picker__free-text"> { __( 'Included in plans' ) } </span>
-						<span className="domain-picker__price-is-paid">
+				{ suggestion.is_free
+					? __( 'Free' )
+					: createInterpolateElement(
+							/* translators: An example would be "Included in paid plan, $15/year", where in mobile view it will only be "$15/year". */
```

Comment:
I feel like translation note is now explaining the application logic instead of translation context.

It can be challenging to figure out what to do with `<price_text>` in different languages.

Reply:
Ah indeed I was hesitant with this implementation too. It was either this or duplication, but I think it's not a bad idea to KISS. https://github.com/Automattic/wp-calypso/pull/43346/files#diff-7047bc1505ee7edb4912dcd6277d5169R85-R100

Reply:
See https://github.com/Automattic/wp-calypso/pull/43346#discussion_r441487453.